src/js/common/schemaform/widgets/CurrencyWidget.jsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+
+export default class CurrencyWidget extends React.Component {
+  onBlur = () => {
+    this.props.onBlur(this.props.id);
+  }
+
+  handleChange = (event) => {
+    const val = event.target.value;
+    if (val === '') {
+      this.props.onChange();
+    } else {
+      // Sometimes we get a string back (e.g. when you enter 10.00)
+      // so we need to parse it to be sure
+      const parsed = parseFloat(val);
+      if (!isNaN(parsed)) {
+        this.props.onChange(parsed);
+      } else {
+        this.props.onChange(val);
+      }
+    }
+  }
+
+  render() {
+    let val = this.props.value;
+    if (typeof val === 'number') {
+      val = val.toFixed(2);
+    }
+
+    return (
+      <input type="number"
+          step=".01"
```

Comment:
I actually had this in there because I had read it was necessary for number inputs to treat decimals correctly, but that's not the case in Chrome. So, I'll do some more digging on that. I would rather use a step of 1.